src/pudl/package_data/meta/datapkg/datapackage.json

```diff
@@ -5421,6 +5522,12 @@
                         "description": "Plant-level sector number, designated by the primary purpose, regulatory status and plant-level combined heat and power status",
                         "format": "default"
                     },
+                    {
+                        "name": "service_area",
+                        "type": "string",
+                        "description": "Service area in which plant is located; for unregulated companies, it's the electric utility with which plant is interconnected",
```

Comment:
Are the contents of this column utility names? How are service areas specified? Is it something that can be referred to elsewhere in the EIA data (e.g. the service territory table in EIA 861?)

Reply:
It looks like it's mostly utility names. We could probably map the service territory based on county and state (available in both EIA860 and EIA861 tables). We could also probably do that for more years than the `service_territory` column is available in 860 (just 2006 and under) due to 861 reporting this out to the current year. How would we handle this?

Reply:
If it's just string names that aren't going to be directly mappable to anything programmatically without more work let's just leave it as is for now.